samples/apps/copilot-chat-app/webapi/DeploymentTemplates/main.bicep

```
@@ -42,7 +43,7 @@ param endpoint string = ''
 param apiKey string = ''
 
 @description('Semantic Kernel server API key - Provide empty string to disable API key auth')
```

Other suggestion from Nilesh (and again in all 4 bicep files:

```suggestion
@description('Semantic Kernel server API key - Generated GUID by default\nProvide empty string to disable API key auth')
```